src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncMethodBuilderMemberCollection.cs

```diff
@@ -341,6 +398,40 @@ internal static bool TryCreate(SyntheticBoundNodeFactory F, MethodSymbol method,
             return true;
         }
 
+        private static MethodSymbol GetCustomCreateMethod(
+            SyntheticBoundNodeFactory F,
+            NamedTypeSymbol builderType)
+        {
+            // The Create method's return type is expected to be buildderType.
+            // The WellKnownMembers routines aren't able to enforce that, which is this method exists.
+            //
+            // TODO: consider removing WellKnownMember.System_Runtime_CompilerServices_AsyncTaskMethodBuilder*__Create
```

Comment:
>  // TODO: consider [](start = 10, length = 19)

In the master branch TODOs should all have associated bug numbers or be removed from source tree.